File: business/src/main/java/movie/dto/MovieDto.java
package movie.dto;

import lombok.Data;

@Data
public class MovieDto {

    private String id;
    private String title;
    private String country;
    private int launch;
    private String category;

    public MovieDto(String id, String title, String country, int launch, String category) {
        this.id = id;
        this.title = title;
        this.country = country;
        this.launch = launch;
        this.category = category;
    }
}

File: frontend/cinema/src/app/services/movie/movie-dto.ts
import {ActorDto} from '../actor/actor-dto';

export class MovieDto {

    id: string;
    title: string;
    country: string;
    launch: number;
    category: string;
    version: number;
    averageVote: number;
    comments: string[] = [];
    actors: ActorDto[] = [];

    constructor(id, title, country, launch, category, version, averageVote, comments, actors) {
        this.id = id;
        this.title = title;
        this.country = country;
        this.launch = launch;
        this.category = category;
        this.version = version;
        this.averageVote = averageVote;
        this.comments = comments;
        this.actors = actors;
    }
}

